microsoft/testsuites/power/power.py

```diff
@@ -0,0 +1,587 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+import time
+from datetime import date
+from typing import List
+
+from assertpy import assert_that
+
+from lisa import (
+    Environment,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    schema,
+)
+from lisa.base_tools import Cat, Uname
+from lisa.features import Disk, StartStop
+from lisa.operating_system import Redhat, Suse, Ubuntu
+from lisa.testsuite import TestCaseMetadata
+from lisa.tools import Blkid, Dmesg, Echo, Fdisk, Sed, Swap, SwapOn
+from lisa.util import LisaException, SkippedException, find_patterns_in_lines
+from lisa.util.logger import Logger
+from lisa.util.shell import wait_tcp_port_ready
+
+
+def check_vm_feature_support_status(min_support_version: str, node: Node) -> bool:
+    if not min_support_version:
+        raise SkippedException("No min support kernel version provided!")
```

Comment:
It looks abnormally called this method, it should raise LisaException, not skip the test case.